yogstation/code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -15,4 +15,10 @@
 	name = "Mindslave Implant (with injector)"
 
 /obj/item/storage/box/syndie_kit/imp_mindslave/PopulateContents()
-	new /obj/item/implanter/mindslave(src)
\ No newline at end of file
+	new /obj/item/implanter/mindslave(src)
+
+/obj/item/storage/box/syndie_kit/imp_greytide
+	name = "Greytide Implant (with injector)"
```

Comment:
```suggestion
	name = "Greytide Implant Injector"
```

Reply:
thats a box and mindslave is doing the same thing 6 lines above that

Reply:
thats a box and mindslave is doing the same thing 6 lines above that